active-response/firewall-drop.sh

```diff
@@ -32,7 +32,8 @@ IP=$3
 PWD=`pwd`
 LOCK="${PWD}/fw-drop"
 LOCK_PID="${PWD}/fw-drop/pid"
-
+IPV4F="$(cat /proc/sys/net/ipv4/ip_forward)"
```

Comment:
hello @ddpbsd, thanks for your attention.
indeed! let me know if this functionality is or not interesting to ossec. If yes, i'll spend more time or more 'ifs' coding a solution.